homeassistant/components/bmw_connected_drive/services.yaml

```diff
@@ -5,45 +5,77 @@
 
 light_flash:
   description: >
-    Flash the lights of the vehicle. The vehicle is identified via the vin
-    (see below).
+    Flash the lights of the vehicle. The vehicle is identified either via its
+    device entry or the VIN. If a VIN is specified, the device entry will be ignored.
   fields:
+    device_id:
+      name: Car
+      description: The BMW Connected Drive device.
+      selector:
+        device:
+          integration: bmw_connected_drive
     vin:
-      description: >
-        The vehicle identification number (VIN) of the vehicle, 17 characters
+      name: VIN
+      description: The vehicle identification number (VIN) of the vehicle, 17 characters.
```

Comment:
Should be "hide" this one by marking it `advanced`?
As a normal user would probably have enough by just using the device.

Reply:
Sounds very good to me, however it doesn't work on my dev environment (rebased to core/dev 10 minutes ago).
Copied it from `homeassistant.reload_config_entry` but also this one does not hide the advanced field `entry_id`.

Is this something that will come up soon? If so, I'll add `advanced: true` for all `VIN` entries.